components/MyTeamMembers.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div class="my-team__members">
+    <article v-for="member of members" :key="member" class="member-card">
```

Comment:
the `:key="member"` meaning your key will be the entire object member. you can use the `member.id` available in your sample data or add index here.

Reply:
@tracnguyen-ms another way to look at this component is for the template to only contain:
```
<article class="member-card"></article>
```

and you could repeat it on the page.

On your page, you could do:
```
<div class="my-team__members">
  <your-component-name v-for="member of members" :key="member.id" :member="member"></your-component-name>
</div>
```

This approach will make your component more re-usable so if you want to use the `member-card` on any part of the website, you can still use it and you can display individual member card

Compared to the current approach where you will always display a group of member cards.

Reply:
I moved to Component like Yuyu's approach